OPHD/UI/Core/Label.cpp

```diff
@@ -50,11 +50,6 @@ void Label::color(const NAS2D::Color& color)
 }
 
 
-int Label::textWidth() const
-{
-	return TXT_FONT->width(text());
-}
```

Comment:
I think this is confusing the `Control` size (`Rectangle`), to which clipping should be done (though probably isn't), and the natural size of the assigned text, which may be larger (which might cause clipping) or smaller (with extra padded space). This value could be used to resize a `Label` `Control` so that it fits snugly around the text that it contains.

Reply:
Label currently always resizes the control based on the size of the text. I wasn't looking to change that behavior here but also wasn't commenting on a preference on clipping or auto resizing. 

I'm unsure what action to take on this comment. Are you looking to have the function textWidth restored?

Reply:
Yes, I think the original code was quite useful, and there is no equivalent for it elsewhere.